core/src/main/scala/org/apache/spark/storage/memory/MemoryStore.scala

```diff
@@ -331,7 +331,15 @@ private[spark] class MemoryStore(
     var unrollMemoryUsedByThisBlock = 0L
     // Underlying buffer for unrolling the block
     val redirectableStream = new RedirectableOutputStream
-    val bbos = new ChunkedByteBufferOutputStream(initialMemoryThreshold.toInt, allocator)
+    val chunkSize = if (initialMemoryThreshold > Int.MaxValue) {
+      logWarning(s"Initial memory threshold of ${Utils.bytesToString(initialMemoryThreshold)} " +
+        s"is too large to be set as chunk size. It is safe to be cap at Int.MaxValue for chunk " +
```

Comment:
How about `Chunk size has been capped to ${Utils.bytesToStringInt.MaxValue)}`instead, in the second sentence? that's more direct.